app/uk/gov/hmrc/claimvatenrolmentfrontend/models/AllocateEnrolmentResponse.scala

```diff
@@ -25,12 +25,15 @@ case object EnrolmentSuccess extends AllocateEnrolmentResponse
 
 case class EnrolmentFailure(errorMessage: String) extends AllocateEnrolmentResponse
 
+case class MultipleEnrolmentsInvalid(errorMessage: String) extends AllocateEnrolmentResponse
+
 object AllocateEnrolmentResponseHttpParser {
 
   implicit object AllocateEnrolmentResponseReads extends HttpReads[AllocateEnrolmentResponse] {
     override def read(method: String, url: String, response: HttpResponse): AllocateEnrolmentResponse =
       response.status match {
         case CREATED => EnrolmentSuccess
+        case CONFLICT => MultipleEnrolmentsInvalid(response.body)
```

Comment:
I think this needs to check for the specific code MULTIPLE_ENROLMENTS_INVALID coming back. There may be other 409s that mean different things.